commands/music/musictrivia.js

```diff
@@ -93,10 +93,41 @@ module.exports = class MusicTriviaCommand extends Command {
       const dispatcher = connection
         .play(
           ytdl(queue[0].url, {
+            // filter: 'audio',
             quality: 'highestaudio',
-            highWaterMark: 1024 * 1024 * 1024
+            highWaterMark: 1 << 25

```

Comment:
this won't solve anything, highWaterMark's value is high enough to reduce stuttering